instrumentation/src/java/AndroidManifest.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  package="org.oppia.android">
+
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <uses-permission android:name="android.permission.INTERNET" />
+  <!-- TODO(#56): Reenable landscape support. -->
+  <application
+    android:name=".instrumentation.OppiaTestApplication"
+    android:allowBackup="true"
+    android:icon="@mipmap/ic_launcher"
+    android:label="@string/app_name"
+    tools:replace="android:name"
+    android:roundIcon="@mipmap/ic_launcher_round"
+    android:supportsRtl="true"
+    android:theme="@style/OppiaTheme">
+    <meta-data
+      android:name="firebase_analytics_collection_deactivated"
+      android:value="true" />
+    <meta-data
+      android:name="firebase_crashlytics_collection_enabled"
+      android:value="false" />
+    <meta-data
+      android:name="automatic_app_expiration_enabled"
+      android:value="false" />
+    <meta-data      android:name="expiration_date"
+      android:value="2020-09-01" />
+
+    <activity
+      android:name=".app.administratorcontrols.AdministratorControlsActivity"
```

Comment:
Why do we need all these activity here?

Reply:
I'm creating a separate application that uses the local dev server so that the user can install and run the test app before writing the tests. which makes it easier to write tests.

Reply:
We really ought to not redefine this. Could you file an issue to remove the redundant declarations here once we support proper manifest merging & add a TODO here for that? It's blocked on finishing the Bazel migration since Gradle doesn't have great manifest merging support.

Reply:
Done,
https://github.com/oppia/oppia-android/issues/3619

Reply:
Add TODO in this file as well.